util/cache/unsafe_lru_cache.go

```diff
@@ -0,0 +1,241 @@
+/*
```

Comment:
@tiancaiamao 
Since the plan cache is maintained in each session locally, we need not to consider the concurrency control over the cache. Thus, I should remove the mutex operations from the original package, which eliminate the unnecessary mutex overhead significantly.